games/Rivencliff__Sunbath.yaml

```
@@ -0,0 +1,76 @@
+name: 'Rivencliff: Sunbath'
+
+brief-description: |
+  A short gay, furry visual novel about Buru, the Bull getting to know better Lunjy, the Lizard in the forest! Available for Windows, macOS, Linux
+
+description: |
+  Rivencliff is gay furry romance visual novel. The story is about Buru, a compassionate bull with healing powers, who has found his place in the peaceful village nestled within a lush valley. 
+
+  The village, a melting pot of cultures and customs, is where he honed his abilities and forged his deep connections with the community. However, the end of his relationship with Taro, his first boyfriend, has left Buru heartbroken and questioning the true nature of love.   
+
+  In this short story "Sunbath", Buru meets Lunjy, one of the barkeepers of the tavern, after he invited Buru to meet him at a secret spot he has in the forest, where Buru and Lunjy get to know each other better and explore all "secret spots" Lunjy has to offer.
+
+  The game offers multiple dialogue choices from time to time and is estimated to take 15-20 minutes to complete.  
+
+
+description-format: markdown
+
+authors:
+  - name: BuruTheBull
+    role:
+      - producer
+  - name: Amethyst Mare
+    role:
+      - artist
+  - name: Maxwell Shiba
+    role:
+      - artist
+  - name: Elise Gordon
+    role:
+      - artist
+  - name: Amalia Martinez
+    role:
+      - artist
+  - name: Teo Frantz
+    role:
+      - artist
+  - name: Laura Sharp
+    role:
+      - producer
+  - name: Blayne Norrick
+    role:
+      - artist
```

```suggestion
authors:
  - name: BuruTheBull
    role: [ producer ]
```
use [flow style](https://www.yaml.info/learn/flowstyle.html) instead